manager/apis/app/v1alpha1/fybrikmodule_types.go

```diff
@@ -236,7 +252,8 @@ type FybrikModule struct {
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
 	// +required
-	Spec FybrikModuleSpec `json:"spec"`
+	Spec   FybrikModuleSpec   `json:"spec"`
+	Status FybrikModuleStatus `json:"status,omitempty"`
 }
```

Comment:
Missing `+kubebuilder:subresource:status` below ``+kubebuilder:object:root=true`? Copy the annotations from one of the other manager types.

Reply:
Added